swap_meet/item.py

```diff
@@ -1,2 +1,34 @@
+import uuid
+
+
 class Item:
-    pass
\ No newline at end of file
+    
+    def __init__(self, id=None, condition=0 ):
+        if id is None:
+            self.id = uuid.uuid4().int
+        else: 
+            self.id = id
```

Comment:
As you get comfortable with python, see if you can turn these types of simple conditionals into ternaries! 

     self.id = uuid.uuid4().int if id is None else id